app/components/FileLoader.js

```diff
@@ -264,14 +277,17 @@ export default class FileLoader extends Component {
 
   renderFileSelection() {
     const store = this.props.store || {};
-    const files = store.files || [];
```

Comment:
I don't think this is referenced anymore?

Reply:
What isn't referenced? allFiles is used a few lines below

Reply:
Might have been looking at wrong commit. Still saw `const files = store.files || [];`